Need to add libraries after object files, or solaris
linker might not see library's symbols to be needed
when added to LDFLAGS.

--- src/Makefile.orig	2011-03-30 13:15:56 +0200
+++ src/Makefile	2011-03-30 13:16:52 +0200
@@ -30,10 +30,10 @@
 	$(CC) $(CFLAGS) -c -o $@ $^
 
 runscript: runscript.o rs-misc.o
-	$(LD) $(LDFLAGS) -o $@ $^ $(LDFLAGS_RS)
+	$(LD) $(LDFLAGS) -o $@ $^ $(LIBS) $(LDFLAGS_RS)
 
 start-stop-daemon: start-stop-daemon.c
-	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $^ $(LDFLAGS_SSD)
+	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $^ $(LIBS) $(LDFLAGS_SSD)
 
 install: $(TARGET)
 	install -m 0755 -d $(DESTDIR)/bin
